hack/tools/scripts/update_trusted_task_bundles.py

```
@@ -0,0 +1,633 @@
+        Returns:
+            True if candidate is newer than current
+        """
+        try:
+            current_parts = [int(p) for p in current.split('.')]
+            candidate_parts = [int(p) for p in candidate.split('.')]
+            return candidate_parts > current_parts
+        except ValueError:
+            return False
+
+
+def fetch_trusted_tasks_data(data_source: str, cache_dir: Optional[str] = None) -> TrustedTasksData:
```

Ruff (0.14.13)

Unused function argument: `cache_dir`

(ARG001)